tests/DCPS/DelayedDurable/rtps_disc.ini

```diff
@@ -12,3 +12,6 @@ ResendPeriod=2
 [transport/the_rtps_transport]
 transport_type=rtps_udp
 use_multicast=0
+heartbeat_period=500
```

Comment:
Does the test work with the default settings?

Reply:
It did pass, but I'd like to keep the adjusted times in there.